README.md

```diff
@@ -203,10 +203,14 @@ The pattern should something like "base-%03d.html"
 ### hocr-wordfreq
 
 ```
-hocr-wordfreq file.html
+hocr-wordfreq [-h] [-i] [-n MAX] [file.html]
```

Comment:
I would rather make the `file.html` without the parenthesis, because we have to specify it directly or pipe it from stdin. But I am not sure how the others are handled here in the list.